packages/firestore/CONTRIBUTING.md

```diff
@@ -43,7 +65,17 @@ yarn test:browser --integration
 # localhost:8080.
 yarn test:browser --integration --local
 
+# Run node tests once.
+yarn test:node
+
 # Run a subset of tests whose names match a filter.
 yarn test:browser --grep 'SortedSet keeps elements in the right order'
 yarn test:node --grep 'SortedSet keeps elements in the right order'
+
+# Run tests against against a Firestore emulator (emulator will be downloaded).
+yarn test:emulator
+
+# Run tests against a Firestore emulator running on localhost:8080.
+yarn test:node:emulator
+yarn test:node:persistence:emulator
```

Comment:
It seems weird that test:emulator and test:node:emulator behave differently wrt the emulator.  I think "node" and "emulator" are separate concerns and adding ":node" shouldn't affect the behavior of ":emulator".

So I think this argues for having ":emulator" and ":running-emulator" or something...  (or ":downloaded-emulator" and ":emulator")